### PR TITLE
Fix test-operator-test-playbooks

### PR DESCRIPTION
**Pull request by sonam1412**
To fix this issue with https://jenkins-cvp-ci.$HOST/job/test-operatorTestPlaybooksTests/160/

### CODE REVIEW

**Comment by sonam1412** [comment]
re-tested version with the change has passed https://jenkins-cvp-ci.apps.ocp-c1.prod.psi.redhat.com/job/test-operatorTestPlaybooksTests/164/

**Review by jsztuka — APPROVED** [review]
lgtm

File: roles/operator_courier_verify/tasks/main.yml [hunk 1]
```diff
@@ -33,4 +31,4 @@
   when:
     - check_verify_result|bool
     - courier_verify_result.rc != 0
-    - not run_upstream|bool
+    - not run_upstream|bool
```
Comment (lslebodn) [inline roles/operator_courier_verify/tasks/main.yml:34]: No new line at the end of file

**Review by samvarankashyap — APPROVED** [review]
LGTM

File: roles/operator_courier_verify/tasks/main.yml (hunk 1 above)
Comment (sonam1412) [reply roles/operator_courier_verify/tasks/main.yml]: added a new line at the end

**Review by lslebodn — APPROVED** [review]
LGTM

**Pull request merged**